spec/libraries/aws_engine_spec.rb

```diff
@@ -68,18 +73,15 @@ def initialize
```

Comment:
__[reek]__
* [UncommunicativeModuleName] has the name 'S3' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Module-Name.md)